build.gradle

```diff
@@ -15,6 +15,7 @@ apply plugin: 'java'
 apply plugin: 'maven'
 apply plugin: "org.sonarqube"
 apply plugin: 'jacoco'
+apply plugin: 'eclipse'
```

Comment:
Could you please remove this change - there is no need to introduce it (as there is no IntelliJ plugin used here either).

Reply:
Removed